### PR TITLE
Update Node Iterator Docs

### PR DESCRIPTION
**Pull request by tshibley**
#### Summary
Small update, just moving the parameters into the correct section + adding the return value for the `createNodeIterator` docs

#### Motivation
Just fixing a small section issue

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/document/createnodeiterator/index.md [hunk 1]
```diff
@@ -57,6 +53,9 @@ None.
 - `filter` {{optional_inline}}
   - : A callback function or an object with an `acceptNode()` method. The function or method will be called for each node in the subtree based at root which is accepted as included by the whatToShow flag to determine whether or not to include it in the list of iterable nodes. The method should return one of `NodeFilter.FILTER_ACCEPT`, `NodeFilter.FILTER_REJECT`, or `NodeFilter.FILTER_SKIP`. See the [Example](#example).
 
+### Return value
```
Comment (OnkarRuikar) [inline files/en-us/web/api/document/createnodeiterator/index.md:56]:
```suggestion
### Return value

```

File: files/en-us/web/api/document/createnodeiterator/index.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ browser-compat: api.Document.createNodeIterator
 ---
 {{APIRef("DOM")}}
 
-Returns a new [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.
+The `Document.createNodeIterator()` creator method returns a newly created [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.
```
Comment (OnkarRuikar) [inline files/en-us/web/api/document/createnodeiterator/index.md:14]:
```suggestion
The `Document.createNodeIterator()` method returns a newly created [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.
```

File: files/en-us/web/api/document/createnodeiterator/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ browser-compat: api.Document.createNodeIterator
 ---
 {{APIRef("DOM")}}
 
-Returns a new [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.
+The `Document.createNodeIterator()` method returns a newly created [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.
```
Comment (teoli2003) [inline files/en-us/web/api/document/createnodeiterator/index.md:14]:
```suggestion
The **`Document.createNodeIterator()**` method returns a new [`NodeIterator`](/en-US/docs/Web/API/NodeIterator) object.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thanks. Some small changes and we can merge this.

File: files/en-us/web/api/document/createnodeiterator/index.md (hunk 3 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/document/createnodeiterator/index.md:14]: **`Document.createNodeIterator()`** 
bold and code got mixed

Comment (teoli2003) [reply files/en-us/web/api/document/createnodeiterator/index.md:14]: Ooops, fixed.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**